src/databricks/labs/ucx/cli.py

```diff
@@ -89,6 +89,20 @@ def create_table_mapping():
     webbrowser.open(f"{ws.config.host}/#workspace{path}")
 
 
+def verify_metastore():
+    """
+    Scans and verifies if a metastore exists for a metastore
+    :param cfg:
+    :return:
+    """
+    ws = WorkspaceClient()
```

Comment:
please extract it as a `VerifyHasMetastore` utility class, that raises a readable exception if metastore is not assigned and call it from:

1. https://github.com/databrickslabs/ucx/blob/main/src/databricks/labs/ucx/runtime.py#L237-L238
2. https://github.com/databrickslabs/ucx/blob/main/src/databricks/labs/ucx/hive_metastore/tables.py#L207-L208

there's no need for a CLI command